### DISCUSSION TITLE
add 'Secret' to 'IP Addresses'

### DISCUSSION THREAD

**Discussion by sgi-ch** [Onemind-Services-LLC/netbox-secrets#111] [Ideas]
Hi. Thanks for development of this Plugin. We're using Netbox mainly as an IPAM without all the features of itself. Is there any plan, to enable the 'Secret' Plugin also to 'IP Addresses' in addition to its existing Apps? Or do i miss something? Thanks.

**Comment by abhi1693** [comment]
This is already possible by adding your desired model to the `apps` config of the plugin. https://github.com/Onemind-Services-LLC/netbox-secrets?tab=readme-ov-file#configuration

**Comment by sgi-ch** [comment]
Thanks for the fast response. I did try that, but then i've an Typo. Do you know is there any list of all "Apps"? Because what i found is not working.

**Reply by abhi1693** [reply]
Here is a sample API from the demo link https://demo.netbox.dev/api/extras/content-types/. You can visit your own netbox for a complete list

**Reply by sgi-ch** [reply]
Thanks! Helpful, but it is not working. Before i open an Issue, i did try to remove - for testing - 'virtualization.virtualmachine' (which is wrote in your README. But is still showing there. Inlcuding restarting.

```
PLUGINS_CONFIG = {
    'netbox-secrets': {
        'apps': ['dcim.device']
    }
}
```
